### PR TITLE
Add repos for EL 9 in updating guide

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Since the latest version is supported on both EL8 and EL9, the repos for EL9 too need to be mentioned in the updating guide.

JIRA: https://issues.redhat.com/browse/SAT-28824

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [X] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/proc_updating-server.adoc [hunk 1]
```diff
@@ -19,6 +19,15 @@ include::snip_warning-maintain-config-noop.adoc[]
...
 . Ensure the {Project} Maintenance repository is enabled:
+** For {EL}9
++
+[options="nowrap" subs="attributes"]
+----
+# subscription-manager repos --enable \
+{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+----
+
+** For {EL}8
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-server.adoc:30]:
```suggestion
** On {RHEL} 8:
```

File: guides/common/modules/proc_updating-server.adoc [hunk 2]
```diff
@@ -19,6 +19,15 @@ include::snip_warning-maintain-config-noop.adoc[]
 .Procedure
 ifdef::satellite[]
 . Ensure the {Project} Maintenance repository is enabled:
+** For {EL}9
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-server.adoc:22]:
```suggestion
** On {RHEL} 9:
```

File: guides/common/modules/proc_updating-smart-proxy.adoc [hunk 3]
```diff
@@ -10,6 +10,15 @@ ifdef::satellite[]
 endif::[]
 . Ensure that the {Project} Maintenance repository is enabled:
 ifdef::satellite[]
+i** For {EL} 9
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-smart-proxy.adoc:13]:
```suggestion
** On {RHEL} 9:
```

File: guides/common/modules/proc_updating-smart-proxy.adoc [hunk 4]
```diff
@@ -10,6 +10,15 @@ ifdef::satellite[]
...
++
+[options="nowrap" subs="attributes"]
+----
+# subscription-manager repos --enable \
+{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+----
+
+** For {EL} 8
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-smart-proxy.adoc:21]:
```suggestion
** On {RHEL} 8:
```

**Comment by AkshayGadhaveRH** [comment]
@evgeni edited disconnected guide as well. Does this look good?

File: guides/common/modules/proc_updating-smart-proxy.adoc [hunk 5]
```diff
@@ -10,6 +10,15 @@ ifdef::satellite[]
 endif::[]
 . Ensure that the {Project} Maintenance repository is enabled:
 ifdef::satellite[]
+** On {EL} 9:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-smart-proxy.adoc:13]:
```suggestion
** On {RHEL} 9:
```

File: guides/common/modules/proc_updating-smart-proxy.adoc [hunk 6]
```diff
@@ -10,6 +10,15 @@ ifdef::satellite[]
...
++
+[options="nowrap" subs="attributes"]
+----
+# subscription-manager repos --enable \
+{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+----
+
+** On {EL} 8:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-smart-proxy.adoc:21]:
```suggestion
** On {RHEL} 8:
```

File: guides/common/modules/proc_updating-server.adoc [hunk 7]
```diff
@@ -19,6 +19,15 @@ include::snip_warning-maintain-config-noop.adoc[]
 .Procedure
 ifdef::satellite[]
 . Ensure the {Project} Maintenance repository is enabled:
+** On {EL} 9:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-server.adoc:22]:
```suggestion
** On {RHEL} 9:
```

File: guides/common/modules/proc_updating-server.adoc [hunk 8]
```diff
@@ -19,6 +19,15 @@ include::snip_warning-maintain-config-noop.adoc[]
...
++
+[options="nowrap" subs="attributes"]
+----
+# subscription-manager repos --enable \
+{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+----
+
+** On {EL} 8:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-server.adoc:30]:
```suggestion
** On {RHEL} 8:
```

File: guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+. In the configuration file, replace the `/root/{Project}-repos` with the extracted location.
+. Use the health check option to determine if the system is ready for update.
+On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} update check \
+--whitelist="check-upstream-repository,repositories-validate"
+----
++
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc:131]:
```suggestion
```

File: guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL9ServerSatelliteServerProjectVersion}]
+name={ProjectName} {ProjectVersion} for RHEL 9 RPMs x86_64
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/layered/rhel9/x86_64/satellite/{ProjectVersion}/os
+enabled=1
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc:38]:
```suggestion
enabled = 1
```

to algin with "sslverify" and all other lines.

Same applies to lines 20, 29, 47, 105, 110, 115, and 120.

File: guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc [hunk 11]
```diff
@@ -0,0 +1,148 @@
+[id="Updating-Disconnected-{project-context}-on-EL8_{context}"]
+= Updating a disconnected {ProjectServer} on {EL} 8
+
+.Procedure on the connected {ProjectServer}
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc:4]: I suggest to add a short introduction to the procedure about the general workflow. the heading and subheading to short succession are slightly confusing to me.

File: guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+[{RepoRHEL8ServerSatelliteMaintenanceProjectVersion}]
+name={ProjectName} Maintenance {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/layered/rhel8/x86_64/sat-maintenance/{ProjectVersion}/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+----
++
```
Comment (maximiliankolb) [inline guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc:53]:
```suggestion
```

**Review by maximiliankolb — COMMENTED** [review]
some minor suggestions

File: guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+* {RepoRHEL8ServerSatelliteMaintenanceProjectVersion}
+. Download the debug certificate of the organization and store it locally at `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
+For more information, see {AdministeringDocURL}Creating_an_Organization_Debug_Certificate_admin[Creating an Organization Debug Certificate] in _{AdministeringDocTitle}_.
+. Create a Yum configuration file under `/etc/yum.repos.d`, such as `_{project-context}-disconnected_.repo`, with the following contents:
++
+[options="nowrap" subs="attributes"]
+----
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/rhel8/8/x86_64/baseos/os
```
Comment (evgeni) [inline guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc:22]: I know this is copied from the old proc, but `_` for italics doesn't work if you don't do `subs="quotes"`. In other places we set `subs="+quotes,verbatim,attributes"` so probably should standardize on everywhere?

**Comment by evgeni** [comment]
LGTM besides the formatting thing

**Review by evgeni — APPROVED** [review]

**Comment by AkshayGadhaveRH** [comment]
@asteflova is this good to be merged?

File: guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} update run \
+--whitelist="check-upstream-repository,repositories-setup,repositories-validate"
+----
+
+include::snip_steps-needs-reboot.adoc[]
+
+.Additional resources
+* To restore the backup of the {ProjectServer} or {SmartProxyServer}, see {AdministeringDocURL}Restoring_from_a_Full_Backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a Backup]
```
Comment (asteflova) [inline guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc:149]:
```suggestion
* To restore the backup of the {ProjectServer} or {SmartProxyServer}, see {AdministeringDocURL}Restoring_from_a_Full_Backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a Backup].
```

File: guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} update run \
+--whitelist="check-upstream-repository,repositories-setup,repositories-validate"
+----
+
+include::snip_steps-needs-reboot.adoc[]
+
+.Additional resources
+* To restore the backup of the {ProjectServer} or {SmartProxyServer}, see {AdministeringDocURL}Restoring_from_a_Full_Backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a Backup]
```
Comment (asteflova) [inline guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc:148]:
```suggestion
* To restore the backup of the {ProjectServer} or {SmartProxyServer}, see {AdministeringDocURL}Restoring_from_a_Full_Backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a Backup].
```

File: guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc [hunk 16]
```diff
@@ -0,0 +1,149 @@
+[id="Updating-Disconnected-{project-context}-on-EL8_{context}"]
+= Updating a disconnected {ProjectServer} on {EL} 8
+
+You can update your disconnected {Project} on {RHEL} 8 by synchronizing the required repositories on the c
+onnected {Project} and syncing the content to the disconnected {Project} using `reposync`.
```
Comment (asteflova) [inline guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc:5]:
```suggestion
You can update your disconnected {Project} on {RHEL} 8 by synchronizing the required repositories on the connected {Project} and syncing the content to the disconnected {Project} using `reposync`.
```

**Review by asteflova — CHANGES_REQUESTED** [review]
The unexpected line break is the only blocking comment. The missing periods are just nitpicks. After you take a look at both, this should be good to merge, especially considering that your new procedures are copied from an old procedure that is already included.

**Review by asteflova — APPROVED** [review]

**Comment by asteflova** [comment]
Merged to "master" and cherry-picked:

* 8e5c3ae1f3..1e706f6cff  3.12 -> 3.12

**Pull request merged**